extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrPhotosImporter.java

```diff
@@ -74,6 +78,18 @@ public FlickrPhotosImporter(
     this.perUserRateLimiter = serviceConfig.getPerUserRateLimiter();
   }
 
+  public FlickrPhotosImporter(
```

Comment:
I think the inverse would be more maintainable; that is: when using overloading for constructors, prefer to have the superset case (the case that has the most params) be the canonical implementation (in this case, that'd be the prior constructor), and have the simpler constructors be a wrapper (where a default is passed to the canonical constructor).

eg:
```java
  public FlickrPhotosImporter(
      AppCredentials appCredentials,
      TemporaryPerJobDataStore jobStore,
      Monitor monitor,
      TransferServiceConfig serviceConfig,
      IdempotentImportExecutor retryingIdempotentExecutor,
      Boolean enableRetrying) {
    this.jobStore = jobStore;
    this.flickr = new Flickr(appCredentials.getKey(), appCredentials.getSecret(), new REST());
    this.uploader = flickr.getUploader();
    this.connectionProvider = new ConnectionProvider(jobStore);
    this.photosetsInterface = flickr.getPhotosetsInterface();
    this.monitor = monitor;
    this.perUserRateLimiter = serviceConfig.getPerUserRateLimiter();
    this.retryingIdempotentExecutor = retryingIdempotentExecutor;
    this.enableRetrying = enableRetrying;
  }

  public FlickrPhotosImporter(
      AppCredentials appCredentials,
      TemporaryPerJobDataStore jobStore,
      Monitor monitor,
      TransferServiceConfig serviceConfig) {
    // DO NOT SUBMIT: kate to spotcheck jon's made-up defaults are correct; eg: is `null` really correct?
    this(appCredentials, jobStore, monitor, serviceConfig, null /*retryingIdempotentExecutor*/, false  /*enableRetrying*/);
  }

```

Reply:
Fixed that, thanks for the explanation!